R/prep2gads.R

```diff
@@ -0,0 +1,91 @@
+prep2GADS <- function (dat, inputList, trafoType = c("scored", "raw"),
```

Comment:
Looks good to me at first glance! I could provide more detailed feedback in May, but as far as I can tell everythings looks very reasonable.

One question though: Is there a specific reason that the importing via `import_raw2()` is left as a separate step? Are there plans on creating a higher order function that combines `prep2GADS()` and `import_raw2()`?